BaseTools/Conf/tools_def.template

```diff
@@ -1947,7 +1947,7 @@ DEFINE GCC_IA32_RC_FLAGS           = -I binary -O elf32-i386          -B i386
 DEFINE GCC_X64_RC_FLAGS            = -I binary -O elf64-x86-64        -B i386    --rename-section .data=.hii
 DEFINE GCC_ARM_RC_FLAGS            = -I binary -O elf32-littlearm     -B arm     --rename-section .data=.hii
 DEFINE GCC_AARCH64_RC_FLAGS        = -I binary -O elf64-littleaarch64 -B aarch64 --rename-section .data=.hii
-DEFINE GCC_RISCV64_RC_FLAGS        = -I binary -O elf64-littleriscv -  B riscv64 --rename-section .data=.hii
+DEFINE GCC_RISCV64_RC_FLAGS        = -I binary -O elf64-littleriscv   -B riscv   --rename-section .data=.hii

```

Comment:
What is the build or functionality error with riscv64

Reply:
```
root@42da53245591:/# riscv64-unknown-elf-objcopy -B riscv64
riscv64-unknown-elf-objcopy: architecture riscv64 unknown
```

Reply:
I have to figure out why it doesn't happen on my build.

Reply:
Ah, I think this error was not triggered for the PPI build.
I think I can trigger it when I build `MdeModulePkg/MdeModulePkg.dsc`.

Reply:
ok. The error happens when build MdeModulePkg/MdeModulePkg.dsc alone, right?

Reply:
Yep. I wanted to build the HelloWorld shell application.

Reply:
I can just add RISCV64 to 
  SUPPORTED_ARCHITECTURES        = IA32|X64|EBC|ARM|AARCH64|`RISCV64` in MdeModulePkg.dsc and MdePkg.dsc then the error you met is gone.
However, I still get build error on MdeModulePkg due to gEfiMdeModulePkgTokenSpaceGuid.PcdDxeIplSwitchToLongMode is not declared in RISC-V arch. This is edk2 problem but our problem. PcdDxeIplSwitchToLongMode is only for x86 system. I will just leave this error there, not dealing with it just now.

Reply:
I updated my compiler toolchain and it's still the case that we need `riscv` not `riscv64` here.

Reply:
@changab You didn't squash this in any commit?

Reply:
No. I will have to check this. just have no time. Will do it soon after I clean up some tasks.
we don't have 'riscv' architecture BTW, we only have either RISCV64 or RISCV32. RISCV32 has been killed from edk2 RISC-V port coz we don't even verify that on QEMU or board.
Give me some time to back on this.